metadata-ingestion/src/datahub/ingestion/source/sql/oracle.py

```diff
@@ -75,6 +78,59 @@ def get_sql_alchemy_url(self):
         return url
 
 
+class OracleInspectorObjectWrapper:
+    """
+    Inspector class wrapper to resolve linear issue
+    https://github.com/datahub-project/datahub/issues/5167
+    """
+
+    def __init__(self, inspector_instance: Inspector):
+        self._inspector_instance = inspector_instance
+        self.log = logging.getLogger(__name__)
+        # tables that we don't want to ingest into the DataHub
+        self.exclude_tablespaces: Tuple[str, str] = ("SYSTEM", "SYSAUX")
+
+    def get_schema_names(self) -> List[str]:
+        logger.debug("OracleInspectorObjectWrapper is in used")
+        s = "SELECT username FROM dba_users ORDER BY username"
+        cursor = self._inspector_instance.bind.execute(s)
+        return [
+            self._inspector_instance.dialect.normalize_name(row[0]) for row in cursor
+        ]
+
+    def get_table_names(self, schema: str = None, order_by: str = None) -> List[str]:
+        """
+        skip order_by, we are not using order_by
+        """
+        logger.debug("OracleInspectorObjectWrapper is in used")
+        schema = self._inspector_instance.dialect.denormalize_name(
+            schema or self.default_schema_name
+        )
+
+        if schema is None:
+            schema = self._inspector_instance.dialect.default_schema_name
+
+        sql_str = "SELECT table_name FROM dba_tables WHERE "
+        if self.exclude_tablespaces:
+            sql_str += "nvl(tablespace_name, 'no tablespace') " "NOT IN (%s) AND " % (
```

Comment:
What's the reason for using implicit string concatenation e.g. `"foo" "bar"` is the same as `"foobar"`, so why break up the strings?

Reply:
I restructured this code, It was SQLAlchemy code, I had updated only the SQL statement on line 113